### PR TITLE
Pia 4417 merge example apps

### PR DESCRIPTION
**Pull request by abolfazlimahdi**

### CODE REVIEW

**Comment by abolfazlimahdi** [comment]
Please notice that I have changed the CI files for example apps.

File: bank-sdk/example-app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -8,42 +8,41 @@
...
+    tools:context=".ui.MainActivity">
 
     <Button
         android:id="@+id/button_start_scanner"
         style="@style/Widget.MaterialComponents.Button.OutlinedButton"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        app:layout_constraintTop_toBottomOf="@id/tv_example_mainActivity"
-        app:layout_constraintBottom_toTopOf="@id/text_gini_capture_version"
+        android:text="@string/start_gini_capture_sdk"
```
Comment (a-szotyori) [inline bank-sdk/example-app/src/main/res/layout/activity_main.xml:18]: [Request] The button should say "Start Gini Bank SDK".

File: bank-sdk/example-app/build.gradle.kts [hunk 2]
```diff
@@ -23,15 +24,15 @@ plugins {
 
 android {
     // after upgrading to AGP 8, we need this (copied from the module's AndroidManifest.xml
-    namespace = "net.gini.android.capture.screen"
+    namespace = "net.gini.android.bank.screen"
```
Comment (a-szotyori) [inline bank-sdk/example-app/build.gradle.kts:27]: [Request] I looked at the Health SDK's example app and it's package name is `net.gini.android.health.sdk.exampleapp`. Let's follow that pattern and use `net.gini.android.bank.sdk.exampleapp` package name for the Bank SDK's example app.

File: bank-sdk/example-app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    package="net.gini.android.capture.screen">
+    package="net.gini.android.bank.screen">
```
Comment (a-szotyori) [inline bank-sdk/example-app/src/main/AndroidManifest.xml:4]: [Request] I looked at the Health SDK's example app and it's package name is `net.gini.android.health.sdk.exampleapp`. Let's follow that pattern and use `net.gini.android.bank.sdk.exampleapp` package name for the Bank SDK's example app.

File: bank-sdk/example-app/src/main/java/net/gini/android/bank/screen/ExampleApp.kt [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import dagger.hilt.android.HiltAndroidApp
+
+
+/**
+ * Main Entry of the application.
+ */
+
+@HiltAndroidApp
+class ExampleApp : MultiDexApplication() {
+
```
Comment (a-szotyori) [inline bank-sdk/example-app/src/main/java/net/gini/android/bank/screen/ExampleApp.kt:13]: [Request] Please remove the empty lines 😉

File: bank-sdk/example-app/src/main/java/net/gini/android/bank/screen/core/di/GiniExampleModule.kt [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+class GiniExampleModule {
+
+    @Singleton
+    @GiniCaptureNetworkServiceRelease
+    @Provides
+    fun bindGiniCaptureNetworkServiceRelease(
+        @ApplicationContext context: Context,
+        logger: Logger
+    ): GiniCaptureDefaultNetworkService {
+        val clientId = context.getString(R.string.gini_api_client_id)
```
Comment (a-szotyori) [inline bank-sdk/example-app/src/main/java/net/gini/android/bank/screen/core/di/GiniExampleModule.kt:37]: [Suggestion] These two methods use duplicated code. You could extract them into private methods. But it's not important. You can also leave it as is.

**Review by a-szotyori — CHANGES_REQUESTED** [review]
Great work! I left some minor requests.

File: bank-sdk/example-app/build.gradle.kts (hunk 2 above)
Comment (abolfazlimahdi) [reply bank-sdk/example-app/build.gradle.kts]: Done.

File: bank-sdk/example-app/src/main/java/net/gini/android/bank/screen/ExampleApp.kt (hunk 4 above)
Comment (abolfazlimahdi) [reply bank-sdk/example-app/src/main/java/net/gini/android/bank/screen/ExampleApp.kt]: Done.

File: bank-sdk/example-app/src/main/java/net/gini/android/bank/screen/core/di/GiniExampleModule.kt (hunk 5 above)
Comment (abolfazlimahdi) [reply bank-sdk/example-app/src/main/java/net/gini/android/bank/screen/core/di/GiniExampleModule.kt]: Done.

**Review by a-szotyori — CHANGES_REQUESTED** [review]
The package name needs one more adjustment.

**Comment by abolfazlimahdi** [comment]
> The package name needs one more adjustment.

I hope it is fixed now. I copied the suggested package name and changed it with "Shift+F6" in the Manifest but I don't know why "sdk" was not included :-/

**Review by a-szotyori — APPROVED** [review]
🎉

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**